x-pack/plugins/observability/public/components/custom_threshold/components/alert_details_app_section/alert_details_app_section.tsx

```diff
@@ -209,14 +184,15 @@ export default function AlertDetailsAppSection({
                 />
               </EuiFlexItem>
               <EuiFlexItem grow={5}>
-                <ExpressionChart
-                  annotations={annotations}
-                  chartType={MetricsExplorerChartType.line}
-                  derivedIndexPattern={derivedIndexPattern}
-                  expression={criterion}
-                  filterQuery={(ruleParams.searchConfiguration?.query as Query)?.query as string}
+                <RuleConditionChart
+                  metricExpression={criterion}
+                  dataView={dataView}
+                  filterQuery={filterQuery}
                   groupBy={ruleParams.groupBy}
-                  hideTitle
+                  annotation={{
+                    timestamp: alert.fields[ALERT_START],
```

Comment:
nit: Maybe `startTimestamp`

Reply:
in Lens, annotation start time is defined with `timestamp` and end time with `endTimestamp` so I kept the same namings for consistency.